### DISCUSSION TITLE
Snapshots Automation

### DISCUSSION THREAD

**Discussion by joaquinjsb** [LibrePDF/OpenPDF#1095] [Ideas]
Hello!
I was wondering if anyone has done any initiative to do snapshot releases, I would be glad to help

**Comment by asturio** [comment]
Ideas are welcome. We deploy the releases using sonatype.

**Comment by asturio** [comment]
I just deployed 2.0.2-SNAPSHOT to https://oss.sonatype.org/content/repositories/snapshots/ manually. See:

https://oss.sonatype.org/content/repositories/snapshots/com/github/librepdf/

Can you check if this is usable. You need to configure your maven or gradle to use this repository too.

**Comment by joaquinjsb** [comment]
Imma do a check this monday!

**Reply by asturio** [reply]
And did you got your project to work with the snapshot?

**Reply by joaquinjsb** [reply]
my bad I didn't get back to you, I confirm that this is working without issues, just had to add snapshot repository:
https://oss.sonatype.org/content/repositories/snapshots/

**Reply by joaquinjsb** [reply]
probably there's an issue on something with the latest snapshots, you are not publishing docs and sources

**Reply by asturio** [reply]
Oh that was propositional. I removed sources and docs for anything, but release. Do you think these are necessary for snapshots?

**Reply by joaquinjsb** [reply]
well, I guess they surely don't do harm, and maybe if there's a new method or class etc, if it's already documented, would be nice to have
